cli/disk_cache.rs

```diff
@@ -146,11 +146,10 @@ mod tests {
 
   #[test]
   fn test_create_cache_if_dir_not_exits() {
-    let cache_location = if cfg!(target_os = "windows") {
-      PathBuf::from(r"C:\deno_dir\foo")
-    } else {
-      PathBuf::from("~/deno_dir/foo")
-    };
+    let temp_dir = TempDir::new().unwrap();
+    let mut cache_location = temp_dir.path().to_owned();
+    assert!(fs::remove_dir(&cache_location).is_ok());
+    cache_location.push("foo");
```

Comment:
If you use `TempDir::new()` then there's no need to remove the directory before the test

Reply:
I did for checking ::new creates directories recursively. The reason to create temp dir is to get safe path that can contain sub directory.